src/lib-rust/src/manager.rs

```diff
@@ -238,16 +238,16 @@ impl UpdateManager {
     }
 
     /// Get a list of available remote releases from the package source.
-    pub fn get_release_feed(&self) -> Result<VelopackAssetFeed, Error> {
+    pub fn get_release_feed(&self, staged_user_id: &str) -> Result<VelopackAssetFeed, Error> {
```

Comment:
The public UpdateManager API probably shouldn't change here: the staged_user_id is known at this point in the UpdateManager locator